nasl/nasl_func.c

```diff
@@ -202,6 +203,16 @@ nasl_func_call (lex_ctxt *lexic, const nasl_func *f, tree_cell *arg_list)
       tree_cell *(*pf2) (lex_ctxt *) = f->block;
 #pragma GCC diagnostic pop
       retc = pf2 (lexic2);
+
+      // ignore when parent and plugin_util got forked
+      if (pu_is_parent ())
+        {
+          g_debug ("%s: setting return value to FAKE_CELL because it is parent "
```

Comment:
during tests, this is being log in the stdout. The log domain is missed in the file